tests/test-amp-blacklist-sanitizer.php

```diff
@@ -102,6 +107,26 @@ public function get_data() {
 				'<a href="http://example.com" target="boom">Link</a>',
 				'<a href="http://example.com">Link</a>',
 			),
+
+			'a_with_href_mailto' => array(
+				'<a href="mailto:email@domain.com">Link</a>',
```

Comment:
This was throwing validation errors in Webmaster Tools as recently as last week. It looks like they've just updated the spec to support it and a few others:


      allowed_protocol: "http"
      allowed_protocol: "https"
      allowed_protocol: "mailto"
      # Whitelisting additional commonly observed third party
      # protocols which should be safe
      allowed_protocol: "sms"
      allowed_protocol: "tel"
      allowed_protocol: "viber"
      allowed_protocol: "whatsapp"

Will update accordingly.